src/solidlsp/language_servers/solargraph.py

```diff
@@ -78,6 +79,13 @@ def _setup_runtime_dependencies(logger: LanguageServerLogger, config: LanguageSe
             raise RuntimeError("Ruby is not installed. Please install Ruby before continuing.") from e
 
         # Check if solargraph is installed
+        # First, try to find solargraph in PATH (includes asdf shims)
```

Comment:
to make it clear that no dependencies are downloaded in that case, could you pls move this code below your addition? Thx!

```
runtime_dependencies = [
            {
                "url": "https://rubygems.org/downloads/solargraph-0.51.1.gem",
                "installCommand": "gem install solargraph -v 0.51.1",
                "binaryName": "solargraph",
                "archiveType": "gem",
            }
        ]

        dependency = runtime_dependencies[0]
```